generators/ruby/sdk/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.0.6] - 2024-02-07
+- Improvement: license files are now specified within the gem config if they are provided
```

Comment:
can we add a small example of the config in `generators.yml`

Reply:
it's not ruby specific though?

Reply:
It's not, but we haven't documented it well enough so anyone who reads this changelog should be able to take advantage of it is my thought